sunpy/map/sources/solo.py

```diff
@@ -46,11 +46,6 @@ def processing_level(self):
             # The level number is prepended by the letter L
             return int(self.meta.get('level')[1:])
 
-    @property
-    def date(self):
```

Comment:
I assume this isn't wildly controversial, but probably should get its own changelog.